modular/manager/manager.go

```diff
@@ -324,11 +325,21 @@ func (m *ManageModular) syncConsensusInfo(ctx context.Context) {
 			}
 		}
 	}
+	storageParams, err := m.baseApp.Consensus().QueryStorageParams(context.Background())
+	if err != nil {
+		log.Errorw("failed to query storage params", "error", err)
+		return
+	}
+	if err = m.baseApp.GfSpDB().SetStorageParams(storageParams); err != nil {
```

Comment:
storage params is not come from DB, should come from Chain, because the param can change by voting.
if store in db, it may make mistake by other developer.

Reply:
Fixed.